gcc/d/ChangeLog

```diff
@@ -1,3 +1,8 @@
+2016-02-22  Eugene Wissner <belka@caraus.de>
+
+	* d-lang.cc (d_init): gcc-6-20160214 removed the second parameter
+	(short_double = false) of build_common_tree_nodes.
```

Comment:
Don't need to reference the gcc snapshot.  The message should be simple and boring, with a complete lack of reasoning behind the change (this is what commit messages are for).